File: prime.java
import java.util.*;
class prime
{
    static final int Max_Size = 1000001;
    static Vector<Boolean>isprime = new Vector<>(Max_Size);
    static Vector<Integer>prime = new Vector<>();
    static Vector<Integer>SPF = new Vector<>(Max_Size);
    static void manipulated(int num)
    {
        isprime.set(0,false);
        isprime.set(1,false);
        for(int i=2;i<num;i++)
        {
            if(isprime.get(i))
            {
                prime.add(i);
                SPF.set(i,i);
            }
            for(int j=0;j<prime.size() && i*prime.get(j)<num && prime.get(j)<=SPF.get(i);j++)
            {
                isprime.set(i*prime.get(j),false);
                SPF.set(i*prime.get(j),prime.get(j));
            }
        }
        
    }
    public static void main(String args [])
    {
        int Sum=0;
        Scanner sc= new Scanner(System.in);
        int t=sc.nextInt();
        
        for(int i =0;i<Max_Size;i++)
        {
            isprime.add(true);
            SPF.add(2);
        }
       
        for(int j=0;j<t;j++)
        {
            int num=sc.nextInt();
            manipulated(num);
        for(int i=0;i<prime.size() && prime.get(i)<=num;i++)
        Sum+=prime.get(i);
        System.out.println(Sum);
        }
        sc.close();
    }
    
}

File: prime.c
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

long Prime(long n)
{
    long flag=0;
    for(long i= 1;i<=n;i++)
    {
        if(n%i==0)
            flag++;
    }
    if(flag==2)
        return n;
    else 
        return 0;
}
long Sum(long num)
{
    long sum = 0;
    for(long j =1;j<=num;j++)
    {
        long u = Prime(j);
        sum+=u;
    }
    return sum;
}
int main() {
long T ;
    long result;
    scanf("%ld",&T);
    while(T--)
    {
        long num;
        scanf("%ld",&num);
         result = Sum(num);
        printf("%ld\n",result);
    }
    
    return 0;
}
